### ISSUE TITLE
Authentication failure! invalid_credentials: OAuth2::Error

### ISSUE THREAD

**Issue by abhinaykumar** [nashby/omniauth-stackexchange#7]
Hey I followed each step and passed required credentials but i am still getting this error

``` json
{"error":{"type":"2","message":"redirect_uri does not match the uri used to create the passed code"}}
```

I have been able to implement omniauth strategy for facebook, google etc. never faced such issue. Is there anything changed from StackExchange api end which is causing this error?

**Comment by gauravtiwari** [comment]
@abhinaykumar Should be fixed in #8

**Comment by nashby** [comment]
@abhinaykumar Hey I know it's been awhile but can you test fix that is on master branch?
